tools/serverpod_cli/bin/util/print.dart

```diff
@@ -1,9 +1,10 @@
 import 'dart:io';
 
+import 'package:colorize/colorize.dart';
 import 'package:super_string/super_string.dart';
 
-/// Print to terminal with word wrap.
-void printww(String string) {
+/// Private print function to terminal with word wrap.
```

Comment:
Moving this to a private method serves no purpose?

Reply:
reverted.